assets/src/dashboard/app/views/storyAnimTool/index.js

```diff
@@ -112,13 +112,16 @@ function StoryAnimTool() {
         saveActiveStoryUpdates(activeStory);
       }
 
+      // Deselect animation
+      setActiveAnimation({});
+
+      // Reset active page
+      setActivePageIndex(0);
```

Comment:
I noticed that we weren't resetting the page index when switching stories.  This causes a crash of our little tool when we're on page 9 of a story and then switch to another story that is 8 pages or less 😱 But now resetting it, we'll always move to the first page when changing stories.